pkg/tests/testdata/expected/views.out

```diff
@@ -49,6 +49,8 @@ SELECT count(compress_chunk(i)) from show_chunks('prom_data.cpu_usage') i;
 (1 row)
 
 \set ON_ERROR_STOP 0
+SET role prom_reader;
```

Comment:
So we connect as prom_writer by default and then set our role to prom_reader right?